packages/block-library/src/post-comments/style.scss

```diff
@@ -45,12 +46,17 @@
 	}
 
 	.comment-meta {
+		font-size: 0.875em;
 		line-height: 1.5;
 		margin-left: -3.25em;
+
+		b {
+			font-weight: normal;
+		}
```

Comment:
I know this feels really un-semantic, but we can't change the markup. 😕 Happy to remove this part if folks feel strongly. But I think it really sticks out if a theme doesn't use bold text elsewhere in the UI.

Reply:
When you say "we can't change the markup", take it you mean in a back-compat sense? To me the part of this that feels unsemantic is the use of the `b` tag, which honestly does feel worth changing (removing?) right at the source through a core patch, that's the only semi-strong feeling I have. Alternatively, would a `font-weight: 500;` change be more palpable? It's _100_ more than the default, but technically still "bolder".

Reply:
Yeah, I guess we _can_ change the markup. I just hadn't really thought about it. 🤔 

In any case, I could change this to `inherit` instead of `normal` and it should usually work fine. Do you think that would be any better than specifying `normal` here?

Reply:
As noted, it's not a super strong feeling, mostly because the semantic concern comes from the fact that the tag reads as "bold", which is kind of beyond interpretation, whereas something like `strong` would leave wiggleroom for how that gets visually interpreted. But we're way in the weeds on nitpickiness and not something I think should block the PR. If anything, we could add a comment that the b tag should be fixed at the source, but even that I'm not sure about.

Reply:
Ok, makes sense. I think we can just move forward with this as is, and change it later on if more folks have concerns about it.